docs/changelog.rst

```diff
@@ -2,7 +2,8 @@
 Changelog
 =========
 
-* :feature:`1022` Include an ``errors`` field in all unsucesful API responses.
+* :feature:`1022` Include an ``errors`` field in all unsuccessful API responses.
+* :feature:`1010` Add ``amount`` and ``target`` to ``EventTransferSentSuccess`` event
```

Comment:
I would suggest adding the changelog on top of the last entry. No need to have them ordered by issue number.

Reply:
When I created the PR it was at the top. I have just rebased since then and thought I'd keep it at the same position as when I originally made the PR.

Reply:
No it's better to have them in the order they go into master. This way we also have an implicit way of checking the order PRs got accepted into master.